rst_files/optimization_solver_packages.rst

```diff
@@ -128,7 +128,7 @@ We have already seen one of the AD packages in Julia
     f(x) = sum(sin, x) + prod(tan, x) * sum(sqrt, x)
     g = (x) -> ForwardDiff.gradient(f, x); # g() is now the gradient
     @show g(rand(20)); # gradient at a random point
-    # ForwardDiff.hessian(f,x2) # or the hessian
+    # ForwardDiff.hessian(f,x) # or the hessian
```

Comment:
```suggestion
    # ForwardDiff.hessian(f,x') # or the hessian
```

The `x2` is probably to indicate that this is a random point. Can change it to `x'`